python/chronos/src/bigdl/chronos/model/tf2/tcn_keras.py

```diff
@@ -0,0 +1,139 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
Please add the same license as https://github.com/intel-analytics/BigDL/blob/32e327133fe8a789890f15a94b5a3a1a3ac70056/python/chronos/src/bigdl/chronos/model/tcn.py#L16-L40 as well